x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/rollup/get_jobs.yml

```diff
@@ -98,129 +98,3 @@ setup:
 
   - match:
       jobs: []
-
----
```

Comment:
Rollup will be removed anyways

Reply:
Rollup hasn't been removed yet though...

Reply:
yes, but with that in mind i don't think anyone will fix that muted test, right?

Reply:
Moved this one to separate pr